File: Vincent OS/chrono.vb
Public Class chrono

    Private Sub chrono_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        If Bureau.Visible = True Then
            Bureau.Button18.Visible = True
        ElseIf Bureau2.Visible = True Then
            Bureau2.Button18.Visible = True
        End If
    End Sub
    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Timer1.Start()
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Timer1.Stop()
    End Sub

    Private Sub Button3_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button3.Click
        Timer1.Stop()
        ProgressBar1.Value = 0
    End Sub

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        ProgressBar1.Increment(2)
        If ProgressBar1.Value = 100 Then
            ProgressBar1.Value = 0
        End If
    End Sub

    Private Sub chrono_FormClosed(ByVal sender As System.Object, ByVal e As System.Windows.Forms.FormClosedEventArgs) Handles MyBase.FormClosed
        If Bureau.Visible = True Then
            Bureau.Button18.Visible = False
        ElseIf Bureau2.Visible = True Then
            Bureau2.Button18.Visible = True
        End If
    End Sub
End Class

File: src/Vincent OS/Chrono.cs
using System;
using System.Windows.Forms;

namespace Vincent_OS
{
    public partial class chrono
    {
        public chrono()
        {
            InitializeComponent();
        }

        private void chrono_Load(object sender, EventArgs e)
        {
            if (My.MyProject.Forms.Bureau.Visible == true)
            {
                My.MyProject.Forms.Bureau.Button18.Visible = true;
            }
            else if (My.MyProject.Forms.Bureau2.Visible == true)
            {
                My.MyProject.Forms.Bureau2.Button18.Visible = true;
            }
        }
        private void Button1_Click(object sender, EventArgs e)
        {
            Timer1.Start();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            Timer1.Stop();
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            Timer1.Stop();
            ProgressBar1.Value = 0;
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            ProgressBar1.Increment(2);
            if (ProgressBar1.Value == 100)
            {
                ProgressBar1.Value = 0;
            }
        }

        private void chrono_FormClosed(object sender, FormClosedEventArgs e)
        {
            if (My.MyProject.Forms.Bureau.Visible == true)
            {
                My.MyProject.Forms.Bureau.Button18.Visible = false;
            }
            else if (My.MyProject.Forms.Bureau2.Visible == true)
            {
                My.MyProject.Forms.Bureau2.Button18.Visible = true;
            }
        }
    }
}